src/index.ts

```diff
@@ -36,4 +41,7 @@ export {
   openAttestationDnsTxt,
   openAttestationEthereumDocumentStoreStatus,
   openAttestationEthereumTokenRegistryStatus,
+  OpenAttestationDnsDid,
+  OpenAttestationDidSignedDocumentStatus,
+  OpenAttestationDidSignedDidIdentityProof,
```

Comment:
To remove ?

Reply:
No, they are suppose to be exported so whoever import can customise their own, but not all of them are in the default set ie `OpenAttestationDidSignedDidIdentityProof`.